astroquery/mast/tesscut.py

```diff
@@ -0,0 +1,322 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+MAST Tesscut
+============
+
+Cutout queries on TESS FFIs.
+
+"""
+
+from __future__ import print_function, division
+
+import warnings
+import time
+import json
+import zipfile
+import os
+
+from io import BytesIO
+
+import numpy as np
+
+import astropy.units as u
+import astropy.coordinates as coord
```

Comment:
it would be more consistent with your tests if you would only import `Angles` here, as that's the only class being used here?

Reply:
Done.